logins-swift/Sync15Passwords/Sync15Logins.xcodeproj/project.pbxproj

```diff
@@ -181,11 +197,11 @@
 			inputPaths = (
 			);
 			outputPaths = (
-				"$(SRCROOT)/../../target/universal/release/libfxa_client.a",
+				"$(SRCROOT)/../../target/universal/release/liblogins_ffi.a",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\ncd \"$SRCROOT\"/../../logins-sql/ffi\nenv -i PATH=$PATH \\\n\"$HOME\"/.cargo/bin/cargo +beta lipo --release\n";
+			shellScript = "\n#cd \"$SRCROOT\"/../../logins-sql/ffi\n#env -i PATH=$PATH \\\n#env -i OPENSSL_DIR=/usr/local/Cellar/openssl/1.0.2p/ \\\n#\"$HOME\"/.cargo/bin/cargo +beta lipo --release --verbose\n";
```

Comment:
Hm... This looks pretty dodgy to me. I think we need to figure out the build/link story with openssl and sqlcipher before landing this.

Reply:
Do we also still need beta? I removed it recently on the FxAClient framework

Reply:
Good call. Let me try and report back